.github/workflows/mobile-compile_time_options.yml

```diff
@@ -110,6 +110,6 @@ jobs:
           --define=envoy_mobile_request_compression=disabled \
           --define=envoy_enable_http_datagrams=disabled \
           --define=google_grpc=disabled \
-          --@envoy//bazel:http3=False \
```

Comment:
Is this removal a mistake, or intentional?

Reply:
intentional - I wanted to make sure H3 builds worked for !yaml for our internal use cases